docs/source/modeling-transactions.rst

```diff
@@ -58,23 +73,9 @@ The output of ``T2`` requires the parameter ``x`` to be a signature that matches
 The key is declared at line 1, and it is prefixed by the keyword ``wif:`` to denote that it is expressed in the Wallet Import Format [#f1]_.
```

Comment:
Typo: 
Now that we are redeeming ``T1``, a transaction with multiple input, ->
Now that we are redeeming ``T1``, a transaction with multiple outputs,